cli/command/manifest/create_list.go

```diff
@@ -34,21 +33,18 @@ func newCreateListCommand(dockerCli command.Cli) *cobra.Command {
 }
 
 func createManifestList(dockerCli command.Cli, args []string, opts annotateOpts) error {
-	// Just do some basic verification here, and leave the rest for when the user pushes the list
 	newRef := args[0]
-	targetRef, err := reference.ParseNormalizedNamed(newRef)
+	targetRef, err := normalizeReference(newRef)
 	if err != nil {
 		return errors.Wrapf(err, "error parsing name for manifest list (%s): %v", newRef)
 	}
+
+	// TODO: why is this here?
```

Comment:
it does some validation on the name that I think we need to keep, e.g. https://github.com/moby/moby/blob/ff4f700f74450018f36d014f3cde0ff1b9c17fb3/registry/config.go#L351